ros_gz_sim/CMakeLists.txt

```
@@ -16,19 +16,21 @@ find_package(CLI11 REQUIRED)
 find_package(ament_cmake REQUIRED)
 find_package(builtin_interfaces REQUIRED)
 find_package(geometry_msgs REQUIRED)
-find_package(gz_math_vendor REQUIRED)
+# find_package(gz_math_vendor REQUIRED)
 find_package(gz-math REQUIRED)
-find_package(gz_msgs_vendor REQUIRED)
+# find_package(gz_msgs_vendor REQUIRED)
 find_package(gz-msgs REQUIRED)
-find_package(gz_sim_vendor REQUIRED)
+# find_package(gz_sim_vendor REQUIRED)
 find_package(gz-sim REQUIRED)
-find_package(gz_transport_vendor REQUIRED)
+# find_package(gz_transport_vendor REQUIRED)
 find_package(gz-transport REQUIRED)
 find_package(rclcpp REQUIRED)
+find_package(rclcpp_action REQUIRED)
 find_package(rclcpp_components REQUIRED)
 find_package(rcpputils REQUIRED)
 find_package(ros_gz_interfaces REQUIRED)
 find_package(std_msgs REQUIRED)
+find_package(simulation_interfaces REQUIRED)
```

update alphabetical order

```suggestion
find_package(simulation_interfaces REQUIRED)
find_package(std_msgs REQUIRED)
```

---

https://github.com/gazebosim/ros_gz/pull/790/commits/a649e5da2c5be781f3169396d0e05933ad71332d